Source/WebCore/dom/ContentVisibilityDocumentState.h

```diff
@@ -54,6 +54,8 @@ class ContentVisibilityDocumentState {
 
     void updateViewportProximity(const Element&, ViewportProximity);
 
+    static void elementContentVisibilityChange(const Element&, const ContentVisibility&, const ContentVisibility&);
```

Comment:
I'm not sure whether there is a strict guideline for this, but I feel like we would typically call this method `didChange` to note that the change occurred.

Reply:
I like it more, fixed.